src/main/java/com/sintern/domain/entity/User.java

```diff
@@ -22,8 +23,14 @@
     @Column(columnDefinition = "uuid")
     UUID id;
 
+    @NotNull(message = "Email is required")
+    @NotBlank(message = "Email is required")
+    @Email(message = "Invalid email format")
     @Column(name = "email", unique = true)
+    @Size(min = 7, max = 74, message = "Email should have between 7 and 74 characters")
```

Comment:
I think the **email** should have a **REGEX** as well.

Reply:
My mistake, I thought the **@Email** annotation was enough.